src/components/Explore/components/TrendsForYou/components/TrendItem/TrendItem.js

```diff
@@ -0,0 +1,43 @@
+import { RiMoreFill } from "react-icons/ri";
+import formatCount from "components/utils/formatCount";
+import "./trendItem.css";
+
+const TrendItem = ({ trend }) => {
+  const isArabic = (str) => {
+    return /[\u0600-\u06FF]/.test(str);
+  };
+
+  const isTrendArabic = isArabic(trend.topic);
+
+  const directionClass = isTrendArabic ? "rtl" : "ltr";
+
+  return (
+    <div className="trend">
+      <div className="trend__info">
+        {trend.category && (
```

Comment:
instead of getting the data from the object each time we can do the followings

const { category, topic, tweetCount } = trend;